test/.eslintrc.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+    "extends": [
+        "../.eslintrc.js"
+    ],
+    "rules": {
+        "no-restricted-imports": "off",
```

Comment:
Would something like this work to prevent people from importing from src?

```suggestion
        "no-restricted-imports": ["error", { "patterns": ["**/src/"] }],
```

I've accidentally imported from src instead of extension.bundle.ts several times and it's annoying because you don't know there's a problem until you push to the build machine and then the tests fail